Source/DevHelper/Template/Filter/Filter.cpp.in

```diff
@@ -83,15 +83,15 @@ void @ClassName@::execute()
   if (getWarningCondition() < 0)
   {
     QString ss = QObject::tr("Some warning message");
-    setWarningCondition(-88888888);
-    notifyWarningMessage(getHumanLabel(), ss, getWarningCondition());
+    setWarningCondition(-88888888); // @@WARNING_CODE@@-88888888@@
+    @@SET_WARNING_CONDITION@@(@@getWarningCondition()@@, ss);
```

Comment:
`setWarningCondition(int)` was removed.

Reply:
Ah yes.  My tool only hit .cpp and .h files and Qt Creator does not include those template files when I do a mass search to check that I replaced them all.  I will fix them.